faros_event.sh

```diff
@@ -399,6 +399,21 @@ function make_artifact_key() {
     fi
 }
 
+function doPullRequestCommitMutation() {
+    if ! [ -z "$has_commit" ] &&
+       ! [ -z "$pull_request_number" ]; then
+            pull_request=$( jq -n \
+                            --arg pull_request_number "$pull_request_number" \
+                            '{
+                                "data_pull_request_uid": $pull_request_number,
+                                "data_pull_request_number": $pull_request_number|tonumber,
+                            }'
+                        )
+            pull_request_commit=$(concat "$pull_request" "$commit_key")
```

Comment:
is `concat` a standard function in sh & bash?

Reply:
No, I defined that here: https://github.com/faros-ai/faros-events-cli/blob/main/faros_event.sh#L628
Should I rename?

Reply:
no need. thanks